appchallenge/reservation.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Reservation</title>
+<meta charset="utf-8">
+<meta name="viewport" content="width-device-width, initial-scale=1">
+<link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css"/>
+<script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+<script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+<!-- Ted Pysh
+	index.html
+	info2439
+	Thoendel
+	9/8/19 
+	-->
+
+
+</head>
+<body>
+<div data-role="page">
+<div data-role="header">
+	<h1>Make A Reservation</h1>
+	<img src="Sage-Bistro-Logo-2017-transparenent-small.png" id="logo">
+</div>
+	<div data-role="header">
+	<a href="index.html" class="ui-btn-right"> Home</a>
+	<a href="survey.html" class="ui-btn"> Take Survey</a>
+	<a href="maps.html" class="ui-btn"> Find Us</a>
+	<a href="menu.html" class="ui-btn"> Menu</a>
+	</div>
+<div role="main" class="ui-content">
+<form action="reservation.php" method="post">
```

Comment:
This is more functionality I did not expect you to tackle! You've done a nice job - and your e-mails about the php make more sense :) You are correct that this won't work without reservation.php - but you've got a nice form to feed it!